### PR TITLE
WIP: Made Prrrs Table Collasible [resolves #83]

### LINKED ISSUE #83: Make Prrrs tables on homepage collapsible.
- make a toggle component to use project-wide

### PR DESCRIPTION
**Pull request by Anasauce**
- created toggle component

### CODE REVIEW

**Comment by deadlyicon** [comment]
OMG squash and polish your PRs before asking me to look at them!!!

**Comment by Anasauce** [comment]
oops! Now we polished our PR 

alt text

File: browser/components/utils/ToggleableSection.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+import Button from '../atoms/Button/index'
+import Icon from '../atoms/Icon/index'
+import './ToggleableSection.sass'
+
+export default class ToggleableSection extends Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      open: this.constructor.initialState
```
Comment (deadlyicon) [inline browser/components/utils/ToggleableSection.js]: you need to defined this if you're going to access it like this. Add

```js
static initialState = true
```
above the constructor

**Pull request merged**